jq-node.js

```diff
@@ -3,7 +3,20 @@
 const fs = require("fs");
 const path = require("path");
 
-const wasmFile = fs.readFileSync(path.join(__dirname, "jq.wasm"));
+const { url } = require("./jq-wasm.url.mjs")
+let wasmPath = path.join(__dirname, "jq.wasm");
+const nextBuildPath = wasmPath.match(/^.+[\\\/]\.next[\\\/]server[\\\/]/)?.[0];
+if (nextBuildPath) {
```

Comment:
i don't think it's immediately obvious that "next" means "Next.js", so might be nice to just put a comment here both saying that and explaining why we need to jump through this hoop (and/or any relevant docs you used to figure it out for future reference).